packages/eas-cli/src/vcs/git.ts

```diff
@@ -56,7 +56,7 @@ export default class GitClient extends Client {
 
   public async hasUncommittedChangesAsync(): Promise<boolean> {
     const changes = await gitStatusAsync({ showUntracked: true });
-    return changes.length !== 0;
+    return changes.length > 0;
```

Comment:
this wasn't necessary to fix the bug but the double negative of "uncommitted" and !== is not well suited for the feeble human brain